yb-voyager/cmd/constants.go

```diff
@@ -16,6 +16,7 @@ limitations under the License.
 package cmd
 
 const (
+	ONE_KB                       = 1024
```

Comment:
Nit: Better way is what followed by the `time` package:
```
const (
   KB = 1024
   MB = 1024 * KB
)
```
Then you can use them like:
```
1 * KB
4 * MB
```